programmers/2일차/두-정수사이의-합.js

```diff
@@ -0,0 +1,13 @@
+function solution(a, b) {
+    const maxAB = Math.max(a,b);
+    const minAB = Math.min(a,b);
+    let sum = 0; 
+    for (let i = minAB; i<=maxAB; i++) {
```

Comment:
```suggestion
    const maxAB = Math.max(a,b);
    const minAB = Math.min(a,b);
    let sum = 0; 
    
    for (let i = minAB; i<=maxAB; i++) {
```

변수 선언 이후에  구문은  한칸 공백을 주셔야 해요